src/Sylius/Bundle/SettingsBundle/Manager/SettingsManagerInterface.php

```diff
@@ -30,10 +27,10 @@
     public function loadSettings($namespace);
 
     /**
-     * Save settings under given namespace.
-     *
      * @param string   $namespace
      * @param Settings $settings
+     *
+     * @throws ValidatorException
```

Comment:
IMO this is wrong, as this is an implementation which throws exception, not interface that decides how to deal with such errors.